web/docs/operators/transformations/top.md

```diff
@@ -5,12 +5,16 @@ Shows the most common values. The dual to [`rare`](rare.md).
 ## Synopsis
 
 ```
-top <field> [--count-field=<count-field>|-c <count-field>]
+top <n> by <field> [--count-field=<count-field>|-c <count-field>]
```

Comment:
The changelog entry still references the old syntax.

Applies to `rare` as well.

Reply:
Fixed.